apps/example/app.json

```diff
@@ -11,11 +11,6 @@
     // disable for now as it causes an Android build error with react-native-auth0
     // See https://github.com/auth0/react-native-auth0/issues/879
     "newArchEnabled": false,
-    "splash": {
-      "image": "./assets/splash-icon.png",
-      "resizeMode": "contain",
-      "backgroundColor": "#ffffff"
-    },
```

Comment:
Removed this one as it's recommended to do it via the config plugin when using `expo-splash-screen`

See https://docs.expo.dev/versions/latest/sdk/splash-screen/#:~:text=You%20can%20also,should%20be%20preferred.